elastic/integrations
# Add an integration for Rubrik with Filesets datastream

## Proposed commit message
We are developing a new integration package for monitoring Rubrik clusters. The PR adds the first data stream to collect information about Rubrik [Filesets](https://docs.rubrik.com/en-us/saas/saas/fs_filesets.html?hl=filesets). Following are the metrics collected:

| Metric                   | Description                                                                 |
|--------------------------|-----------------------------------------------------------------------------|
| Archive Storage          | Amount of archived storage data, in bytes.                                |
| Archived Snapshot Count  | Number of currently archived fileset snapshots.                           |
| Local Storage            | Amount of local storage used, in bytes, by the fileset.                   |
| Snapshot Count           | Current number of fileset snapshots.                                      |



### Note for reviewers:
Please ignore the fact the fact the changes don't contain the docs yet. That will be added for the entire package later once we have a few data streams merged. 

## Checklist


## Author's Checklist



## How to test this PR locally
- Spin up the Elastic stack and point your `elastic-package` to the running Kibana instance
- Build the package locally and install the package using `elastic-package install --zip rubrik-0.1.0.zip`
- Observe the metrics in Kibana > Discovers section under the `rubrik_filesets` dataset


## Related issues
- Closes https://github.com/elastic/obs-integration-team/issues/115

## Screenshots
![Uploading image.png…]()
